spec/controllers/api/sequences/sequences_index_spec.rb

```diff
@@ -11,7 +11,7 @@
     it 'lists all sequences for a user' do
       sign_in user
       sequences = FactoryGirl
-                    .create_list(:sequence, 2, user: user)
+                    .create_list(:sequence, 2, device: user.device)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.